### DISCUSSION TITLE
How to get VueForm Schema JSON content?

### DISCUSSION THREAD

**Discussion by yangbin990** [vueform/vueform#379] [Questions]
Hi All,

I have two questions related with VueForm JSON Schema.
Question 1
Anyone has the idea how to get the VueForm JSON content? So it can be saved into database.
Question 2
I also want to understand what the JSON schema looks like for passing the onChange event through the JSON schema file. 
But if the first question 1 is answered, then actually I would be able to figure the question 2 by saving a form.

Thanks 
Bin

**Comment by adamberecz** [comment]
Hi, if you define your form inline, there's no way to export the `schema`. That only works the other way-around.

Events can be used like this in `schema`:
```vue
<script>
import { Vueform, useVueform } from '@vueform/vueform'

export default {
  mixins: [Vueform],
  setup: useVueform,
  data: () => ({
    vueform: {
      schema: {
        element: {
          type: 'text',
          on{EventName}() {
            // ...
          }
        }
      }
    }
  })
}
</script>
```
